Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM nginx:1.13-alpine
+
+COPY nginx/ /etc/nginx
+COPY build/* /usr/share/nginx/www/
```

Comment:
you can use COPY build/*/** /usr/share/nginx/www/ to avoid creating additional COPY

Reply:
Thank you, but it isn't working.